src/codegen/sys/lib_.rs

```diff
@@ -182,6 +183,7 @@ fn generate_interfaces_structs<W: Write>(w: &mut W, interfaces: &[&library::Inte
 fn generate_records<W: Write>(w: &mut W, env: &Env, records: &[&library::Record]) -> Result<()> {
     try!(writeln!(w, ""));
     for record in records {
+        if record.c_type == "GValue" { continue; }
```

Comment:
I don't think it is. Removing these two definitions doesn't break `glib-sys` for me.